src/pass/storage_rewrite.cc

```diff
@@ -569,12 +568,22 @@ class StoragePlanRewriter : public IRMutator {
                     make_const(sz.type(), alloc_type.lanes() - 1)) /
                   make_const(sz.type(), alloc_type.lanes());
             }
+            // transform to bits
```

Comment:
Read the if condition block before this, this function need to be updated to use the bit style logic you are now proposing

Reply:
since we will deal with round up later, so i just remove this if, correct me if i'm wrong